test/salt/pillar/postgres.sls

```diff
@@ -181,7 +181,7 @@ postgres:
         public:
           owner: localUser
       # enable per-db extension
-      {%- if grains.os_family == 'Debian' and grains.osfinger != 'Debian-8' %}
+      {%- if grains.os_family == 'Debian' and salt['grains.get']('osfinger') != 'Debian-8') %}
```

Comment:
This one at the end and later are superfluous too :smiley:

Reply:
I nailed them all now. Cmon *Lint* !!